docs/manual/java/gettingstarted/WhatsNext.md

```diff
@@ -0,0 +1,22 @@
+# What's Next?
+
+After working through "Getting started with Lagom", follow the path that make the most sense to you:
+
+Learn how to integrate with Eclipse:
+
+* [[Import an existing Maven project into Eclipse|EclipseMavenInt]] 
+* [[Create a new Eclipse project using the Lagom Maven archetype|EclipseMavenNew]]
+* [[Import an existing sbt build into Eclipse|EclipseSbt]]
+    
+Learn how to integrate with Intellij:
+
+* [[Import an existing Maven project into Intellij|intellijMaven]]
+* [[Import an existing sbt build into Intellij|IntellijSbtJava]]
+    
+Learn more about Lagom:
+
+* [[Core concepts|CoreConcepts]]
+* [[Using the service locator|ServiceLocator]]
+* [[Using the message broker|MessageBrokerAPI]]
+* [[Using publish-subscribe|PubSub]]
+* [[Using persistent entities|PersistentEntity]]
```

Comment:
The "Next" link at the bottom of this page is labelled "Lagom core concepts", but goes to the "Lagom design philosophy" page and not the "Core concepts' page. I think this has to do with the order of the pages in `index.toc`